src/types/models.ts

```diff
@@ -52,14 +53,14 @@ export type ThermostatDeviceProperties = CommonDeviceProperties & {
   temperature_celsius: number
   temperature_fahrenheit: number
   current_climate_setting: {
-    hvac_mode_setting: "off" | "heat" | "cool" | "heatcool"
```

Comment:
It may be wise to have an `HvacModeSetting` type here as well, we have it in public types

https://github.com/seamapi/seam-connect/blob/main/public-types/unstable/models/capability-properties/thermostat.ts#L5

`type HvacModeSetting = "off" | "heat" | "cool" | "heat_cool"`